src/main/java/org/terasology/sensors/PhysicalSensorComponent.java

```diff
@@ -4,9 +4,20 @@
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.network.Replicate;
 
+/**
+ * A sensor located physically in the world, which can detect entities when activated.
```

Comment:
Does it need to be "in the world"? I don't see a location component here, but I also might be missing something...